src/assets/styles/_main.scss

```diff
@@ -0,0 +1,74 @@
+@import 'node_modules/bootstrap/scss/bootstrap';
+
+:root {
+  --header-height: 56px;
+}
+
+body,
+html {
+  margin: 0;
+  padding: 0;
+  height: 100vh;
+}
+
+.container-height {
+  height: calc(100vh - var(--header-height));
+}
+
+.mw-500 {
+  max-width: 500px;
+}
+
+.float-right {
+  float: right;
+}
+
+.h-200 {
+  min-height: 200px;
+  max-height: 200px;
+}
+
+.overflow-auto {
+  overflow: auto;
+}
+
+.c-pointer {
+  cursor: pointer;
+}
+
+.resize-none {
+  resize: none;
+}
+
+.mr-2 {
+  margin-right: ($spacer * 0.5) !important;
+}
+
+.btn-primary {
+  color: #fff;
```

Comment:
No hard-coding of colors - the whole point of using BS overrides is not having to do this - refer to the bludot-community-frontend zip sent to you
There should be a separate variables.scss file where we import the bootstrap css and then override it